### PR TITLE
CASMINST-4431 - updated prerequisites page with explicit prescriptve …

### PR DESCRIPTION
**Pull request by alexanderkingh**
…instructions to determine if switch configs have been upgraded and if not, to go follow the steps and come back to this section of the docs.

## Summary and Scope

_Summarize what has changed. Explain why this PR is necessary. What is impacted? Is this a new feature, critical bug fix, etc?_

_Is this change backwards incompatible, backwards compatible, or a backwards compatible bugfix?_

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves [issue id](issue link)
* Change will also be needed in `<insert branch name here>`
* Future work required by [issue id](issue link)
* Documentation changes required in [issue id](issue link)
* Merge with/before/after `<insert PR URL here>`

## Testing

_List the environments in which these changes were tested._

### Tested on:

  * `<development system>`
  * Local development environment
  * Virtual Shasta

### Test description:

_How were the changes tested and success verified? If schema changes were part of this change, how were those handled in your upgrade/downgrade testing?_

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)?
- Were continuous integration tests run? If not, why?
- Was upgrade tested? If not, why?
- Was downgrade tested? If not, why?
- Were new tests (or test issues/Jiras) created for this change?

## Risks and Mitigations

_Are there known issues with these changes? Any other special considerations?_

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [ ] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

**Review by lukebates123 — APPROVED** [review]

File: upgrade/1.2/Stage_0_Prerequisites.md [hunk 1]
```diff
@@ -174,6 +174,22 @@ ncn-m001# curl -H "Authorization: Bearer ${TOKEN}" -k -L -X POST 'https://api-gw
...
+  1. After logging into each switch, you will see output like the below in your prompt if the switches have a CANU generated config for CSM 1.2 in place.
+
+   ```
+##################################################################################
+    # CSM version:  1.2
+    # CANU version: 1.3.2
+##################################################################################
+   ```
+
+  3. **`Warning:`** If the switch does NOT show like the above output, stop, and go perform the switch config upgrade steps in the [Upgrade Switches From 1.0 to 1.2 Preconfig](../../operations/network/managemenet_network/upgrade.md#upgrade-switches-from-10-to-12-preconfig). Once you've completed each step, return to this page.
```
[the remark this answers is not in the document]
Comment (alexanderkingh) [reply upgrade/1.2/Stage_0_Prerequisites.md]: Just updated the wording.

**Review by jasonrouault — APPROVED** [review]

File: upgrade/1.2/Stage_0_Prerequisites.md [hunk 2]
```diff
@@ -196,7 +213,7 @@ ncn-m001# export SW_ADMIN_PASSWORD=sw1tCH@DM1Np4s5w0rd
 > For example:
 >
 > ```bash
-> ncn-m001# export NEXUS_PASSWORD=cu$t0m@DM1Np4s5w0rd
+> ncn-m001# export NEXUS_PASSWORD=PutYourOwnPasswordHered
```
Comment (nrockershousen) [inline upgrade/1.2/Stage_0_Prerequisites.md:216]: There is an extra 'd' character at the end of the placeholder value

File: upgrade/1.2/Stage_0_Prerequisites.md [hunk 3]
```diff
@@ -174,6 +174,23 @@ ncn-m001# curl -H "Authorization: Bearer ${TOKEN}" -k -L -X POST 'https://api-gw
 
 ## Stage 0.3 - Upgrade Management Network
 
+#### Verify if Switches have 1.2 Configuration in place.
+
+  1. Login to each management switch `ssh admin@1.2.3.4`
+
+  1. After logging into each switch, you will see output like the below in your prompt if the switches have a CANU generated config for CSM 1.2 in place.
+
+   ```
```
Comment (nrockershousen) [inline upgrade/1.2/Stage_0_Prerequisites.md:183]: Indent this code block so it is under the preceding step.

**Comment by mharding-hpe** [comment]
backport release/1.2.5

**Comment by mharding-hpe** [comment]
backport release/1.2.5

**Pull request merged**